index.d.ts

```diff
@@ -0,0 +1,9 @@
+declare class SemanticReleaseError extends Error {
+    public name: string;
+    public message: string;
```

Comment:
The `Error` [typings](https://github.com/Microsoft/TypeScript/blob/004dc8960069c0217a2e804ffc19fc02917ab043/lib/lib.es5.d.ts#L962-L966) specify `message` and `name` already, so you don't need to respecify them here.

Reply:
Good point.